dist/index.html

```diff
@@ -345,7 +345,7 @@ <h3 class="section-subheading text-muted">Do you want to start using OGC API Fea
                                     <a href="https://github.com/qgis/QGIS/"><i class="fab fa-github"></i></a>
                                 </td>
                                 <td title="Docker image">
-                                    <a href="https://hub.docker.com/r/kartoza/qgis-server"><i class="fab fa-docker"></i></a>
+                                    <a href="https://hub.docker.com/r/opengisch/qgis-server"><i class="fab fa-docker"></i></a>
```

Comment:
Thank You @mbernasocchi ! The index.html is generated from index.pug. Could you make the change here? 🙏🏽 

https://github.com/opengeospatial/ogc-features-website/blob/8f56cc2acf3f2e9e1c5323c870ee14e7e0cd8ee8/src/pug/index.pug#L302